lib/valueflow.cpp

```diff
@@ -2429,8 +2429,9 @@ static Analyzer::Action valueFlowForwardVariable(Token* const startToken,
         TokenList* const tokenlist,
         const Settings* const settings)
 {
+    auto aliases = getAliasesFromValues(values);
     return valueFlowForwardVariable(
-               startToken, endToken, var, std::move(values), getAliasesFromValues(values), tokenlist, settings);
+               startToken, endToken, var, std::move(values), std::move(aliases), tokenlist, settings);
```

Comment:
I don't understand why this would be better. you copy the vector anyway.

Reply:
`values` is already moved before the call which takes it as parameter...